src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj

```diff
@@ -126,6 +128,7 @@
     <PackageReference Include="Microsoft.ServiceHub.Client" Version="$(MicrosoftServiceHubClientVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Debugger.Engine-implementation" Version="$(MicrosoftVisualStudioDebuggerEngineimplementationVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Designer.Interfaces" Version="$(MicrosoftVisualStudioDesignerInterfacesVersion)" />
+    <PackageReference Include="Microsoft.VisualStudio.Extensibility.Testing.SupportFiles" GeneratePathProperty="true" Version="$(MicrosoftVisualStudioExtensibilityTestingSupportFilesVersion)" PrivateAssets="all" />
```

Comment:
How was this package produced?

Reply:
Entirely manually. It contains only reference assemblies.

Reply:
I would need a timeline on when the public API will be available to decide if I am ok paying this additional engineering cost.

Reply:
@jmarolf The technical debt is strictly reduced by this change, even for the case where no public API is ever added.

Reply:
We at least need documentation on 
1. how this package was created
2. how to update it when necessary
3. an explanation of what security validation has been done on this dependency

Reply:
> how this package was created

It was manually created for use by microsoft/vs-extension-testing.

> how to update it when necessary

There is no automation in place for updating this package. A new one would need to be created by hand if ever necessary.

> an explanation of what security validation has been done on this dependency

There is no executable code or build logic in this dependency. It's a Zip file containing reference assemblies in a location specifically designed to avoid any special treatment by NuGet tools.